src/Microsoft.Health.Fhir.SqlServer.UnitTests/Features/ChangeFeed/SqlServerFhirResourceChangeDataStoreTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Health.Fhir.SqlServer.Features.ChangeFeed;
+using Microsoft.Health.SqlServer;
+using Microsoft.Health.SqlServer.Configs;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.SqlServer.UnitTests.Features.ChangeFeed
+{
+    /// <summary>
+    /// Unit tests to validates input parameters of a GetRecordsAsync function.
+    /// </summary>
+    public class SqlServerFhirResourceChangeDataStoreTests
+    {
+        private readonly SqlServerFhirResourceChangeDataStore resourceChangeDataStore;
+
+        public SqlServerFhirResourceChangeDataStoreTests()
+        {
+            var config = new SqlServerDataStoreConfiguration { ConnectionString = string.Empty };
+            var connectionStringProvider = new DefaultSqlConnectionStringProvider(config);
+            var connectionFactory = new DefaultSqlConnectionFactory(connectionStringProvider);
+            resourceChangeDataStore = new SqlServerFhirResourceChangeDataStore(connectionFactory, NullLogger<SqlServerFhirResourceChangeDataStore>.Instance);
+        }
+
+        [Fact]
+        public async Task GivenTheStartIdLessThanOne_ExceptionShouldBeThrown()
```

Comment:
Test names are usually "Arrage_Act_Assert" which we use as "Given_When_Then"

Reply:
Renamed the tests.